custom/_legacy/hsph/reports/field_management.py

```diff
@@ -34,10 +38,6 @@ def numeric_cell(val):
 def short_date_format(date):
     return date.strftime('%d-%b')
 
-        
-def datestring_minus_days(datestring, days):
-    date = datetime.datetime.strptime(datestring[:10], '%Y-%m-%d')
-    return (date - datetime.timedelta(days=days)).isoformat()
```

Comment:
The exact same function was redefined here, so I just made it call the first one